components/generators/kubernetes/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 search_paths = args.get("search_paths")
 
-inv = inventory()
+inv = inventory(lazy=True)
```

Comment:
ideally, we'd want to not use lazy

Reply:
I intend to refactor it heavily to move to the new BaseClass, as part of that we might achieve this as well